packages/core/src/tracing/idleSpan.ts

```diff
@@ -108,6 +109,20 @@ export function startIdleSpan(startSpanOptions: StartSpanOptions, options: Parti
   const previousActiveSpan = getActiveSpan();
   const span = _startIdleSpan(startSpanOptions);
 
+  function _endSpan(timestamp: number = timestampInSeconds()): void {
+    // Ensure we end with the last span timestamp, if possible
+    const spans = getSpanDescendants(span).filter(child => child !== span);
+
+    const latestEndTimestamp = spans.length ? Math.max(...spans.map(span => spanToJSON(span).timestamp || 0)) : 0;
```

Comment:
Just a small concern. I think relay drops all events where a span has a end timestamp smaller than the start timestamp. I am worried that defaulting to 0 may produce such cases by accident. Should we default to `timestamp` when `spans` is empty? I know below we are checking implicitly for thether latest end timestamp is truthy but this is a bit too implicit for my taste :P

Reply:
Yeah, so my thinking was that this would be captured below by the check, but I can make this a bit nicer/better understandable :D